src/java.base/share/classes/jdk/internal/access/JavaLangAccess.java

```diff
@@ -453,6 +453,8 @@ public interface JavaLangAccess {
      */
     String join(String prefix, String suffix, String delimiter, String[] elements, int size);
 
+    String concat(char prefix, String value, char suffix);
```

Comment:
Please add a line of comment, like `Concatenation of prefix and suffix characters to a String for early bootstrap.` So in the future, if we have other ways to fast concat in early bootstrap, people will know this hook is safe to remove.